packages/web/src/components/leaderboard/leaderboard-table/web-leaderboard-table/WebLeaderboardTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { LEADERBOARD_TD_WIDTH } from "@constants/skeleton.constant";
+import { LeaderModel } from "@models/leaderboard/leader-model";
+import LeaderboardTableRow from "../../leaderboard-table-row/LeaderboardTableRow";
+
+export default function WebLeaderboardTable({
+  me,
+  leaders,
+  isMobile,
+}: {
+  me?: LeaderModel;
+  leaders: LeaderModel[];
+  isMobile: boolean;
+}) {
+  return (
+    <>
+      {me && (
+        <LeaderboardTableRow
+          item={me}
+          tdWidths={LEADERBOARD_TD_WIDTH}
+          isMobile={isMobile}
+          me={true}
+        />
+      )}
```

Comment:
Is there a reason you didn't pass `me` directly?

Reply:
the same poor naming, I will fix the variable names as well.